### ISSUE TITLE
Windows 7 - Import Error: no module named 'ant'

### ISSUE THREAD

**Issue by xaviergm** [Tigge/antfs-cli#153]
Hi,
I am helping a guy get sorted in windows 7 with antfs-cli in order to sync his 910xt after one of the activities got corrupted. Apparently this is the only way.
We managed to set it up completely, but when trying to sync he gets the following error:
![img_1009](https://cloud.githubusercontent.com/assets/10818834/20636097/e476327a-b34d-11e6-9eb5-4ecb35c76397.JPG)

I'm going to try and get the text version of the screenshot...

**Comment by Tigge** [comment]
I haven't run this on Windows at all I'm afraid and it is not coded to work on Windows really. The underlying openant library should theoretically work on Windows but I don't think this have been attempted.

**Comment by xaviergm** [comment]
> The underlying openant library should theoretically work on Windows but I don't think this have been attempted.
Is openant for Linux meant to act as the ANT+ Driver for then running the antfs-cli on top? I believe the problem is that there are no such thing as kernel modules (at least not with the same architecture, I guess) in Windows that can be spawned just like in Linux.
> I haven't run this on Windows at all I'm afraid and it is not coded to work on Windows really.
I can see that :joy: Let me investigate a bit... otherwise I will need to tell the guy that he has to factory reset his 910xt, losing all the activities after the corrupted one...

**Comment by xaviergm** [comment]
One last quick think: what is the Python version compatibility? The guy installed the latest (3.5). Is your software compatible with Python 3 or just with Python 2?

**Comment by Tigge** [comment]
Openant is an ANT/ANT-FS/ANT+ library - ANT is the low lever protocol and ANT-FS is implemented on top of ANT and is what gets your activities of the sports watch. Openant talks directly to the USB stick through one of several supported usb-libraries.

Python version 2.7 or 3.5 - 3.7 is fine. You could try to run Linux in a VM, could be an option?

**Comment by xaviergm** [comment]
Yep, I'm looking at VirtualBox just now... what are the minimum requirements you'd recommend for this? Any distro in particular that would make the job easier than others?

**Comment by Tigge** [comment]
Nah, should be about the same. I prefer a Debian based distribution - but that's just me.
